SensusUI/PromptForInputsPage.cs

```diff
@@ -202,7 +211,7 @@ public PromptForInputsPage(InputGroup inputGroup, int stepNumber, int totalSteps
 
             Button nextButton = new Button
             {
-                HorizontalOptions = LayoutOptions.FillAndExpand,
+                HorizontalOptions = LayoutOptions.CenterAndExpand,
```

Comment:
I'm not sure why `CenterAndExpand` is used. Do `Start` and `End` work for previous and next, respectively?